### PR TITLE
Admin API SDK: Add List Identities Route

### PR DESCRIPTION
**Pull request by MH321Productions**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: AdminApi.Sdk/Endpoints/Identities/IdentitiesEndpoint.cs
Comment (MH321Productions) [reply AdminApi.Sdk/Endpoints/Identities/IdentitiesEndpoint.cs:17]: > Wtf, does the `../../` actually work in a URL? 😮

It actually does 🤣

Comment (tnotheis) [reply AdminApi.Sdk/Endpoints/Identities/IdentitiesEndpoint.cs:17]: Wow, I did not know this was a thing. 🤣

File: BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs
Comment (daniel-almeida-konkconsulting) [reply BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs:91]: Keep in mind there are other differences between the response payloads.
OData, for example, adds a `@odata.context` property (e.g., `"@odata.context":"http://localhost/odata/$metadata#Identities"`)

Comment (MH321Productions) [reply BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs:91]: Well, do I need to extract those?

Comment (daniel-almeida-konkconsulting) [reply BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs:91]: Ideally, yes.
If what we are striving for is an exact match between both response, we should only have a result property for success cases. This is especially true when considering that the schema validation is made against the Raw Content of the response (hence an odata.context property will cause it to fail).

Comment (MH321Productions) [reply BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs:91]: It doesn't fail. My current class looks like this:

```c#
public class ODataResponse<TResult>
{
    public TResult? Value { get; set; }

    public ApiResponse<TResult> ToApiResponse(HttpStatusCode status) => new() { Result = Value, Status = status };
}
```

I tested it and it works flawlessly. But if you want, I can make this class a wrapper with the context and the value and put this in the ```ApiResponse``` as ```ApiResponse<ODataReponse<T>>```.

Comment (MH321Productions) [reply BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs]: Note to outside readers: Daniel and me had a misunderstanding and we resolved it privately.

File: BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs
Comment (tnotheis) [inline BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs:26]: These are not used

**Review by tnotheis — CHANGES_REQUESTED** [review]
I let @daniel-almeida-konkconsulting do the final review. As soon as he approved I will give my approval as well (this is required because I'm code owner)

File: BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs
Comment (MH321Productions) [reply BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs]: Touché

File: AdminApi.Sdk/Endpoints/Identities/Types/Requests/CreateIdentityRequest.cs
Comment (tnotheis) [inline AdminApi.Sdk/Endpoints/Identities/Types/Requests/CreateIdentityRequest.cs:9]: Why didn't you keep the byte[] type? Imo it's better, because in case of a string, you don't know that it has to be base64 encoded.

File: AdminApi.Sdk/Services/AccountController.cs
Comment (tnotheis) [inline AdminApi.Sdk/Services/AccountController.cs:1]: Why is this class part of a PR with the name "Add List Identities Route"?

**Review by tnotheis — CHANGES_REQUESTED** [review]

**Pull request merged**
